app/models/corpus_item_model.py

```diff
@@ -1,7 +1,17 @@
-from pydantic import BaseModel
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel, Field
 
 
 class CorpusItemModel(BaseModel):
     id: str
     topic: str = None
     publisher: str = None
+
+    ranked_with_engagement_updated_at: Optional[datetime] = Field(
```

Comment:
I'm a bit out of context but recAPI doesn't look like the right place to monitor issues in our data pipelines. Approving anyway

Reply:
:thinking: Maybe I should've considered asking Braun for review, because he and I discussed how we could implement our SLIs. I'll think about that next time.

Reply:
We want the recommendation freshness SLI to cover the entire pipeline responsible for Thompson sampling. recAPI is the last link in the chain, so we need to emit some data here to know which engagement data was used.